.github/workflows/docker_images.yml

```diff
@@ -0,0 +1,71 @@
+name: "Publish Docker Images"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version tag (e.g., v1.0.0)'
+        required: true
+      specific_image:
+        description: 'Build only this image (leave empty to build all images)'
```

Comment:
maybe add a list of available images to build individually (if description max size allows it)